pkgs/applications/video/obs-studio/plugins/obs-vkcapture.nix

```diff
@@ -22,8 +23,27 @@ stdenv.mkDerivation rec {
     hash = "sha256-yaN0am24p9gC+s64Rop+jQ3952UOtZund/KttnVxP48=";
   };
 
+  cmakeFlags = lib.optionals stdenv.isi686 [
+    "-DBUILD_PLUGIN=OFF"
+  ];
```

Comment:
Please add a comment why this is disabled.

Reply:
Ideally ideally we would put this behind a flag and only build without plugin when building for obv-vkcapture32 but nobody uses i686-linux to play VK games, much less capture them.